src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -116,29 +116,21 @@ using AcceptorPtr = std::unique_ptr<grpc_tcp_server_acceptor, AcceptorDeleter>;
 class Chttp2ServerListener : public Server::ListenerInterface {
  public:
   static grpc_error_handle Create(Server* server, grpc_resolved_address* addr,
-                                  const ChannelArgs& args,
-                                  Chttp2ServerArgsModifier args_modifier,
```

Comment:
I think we can also remove the definition of `Chttp2ServerArgsModifier` from chttp2_server.h.

The declaration of `Chttp2ServerAddPort()` can also be removed completely from the .h file, since it looks like it's never used externally.